README.md

```diff
@@ -57,10 +104,6 @@ The chmod permissions are then reverted.
           script: 'chmod 755 -R $(Build.ArtifactStagingDirectory)'
       displayName: "Revert chmod permissions (ArtifactStagingDirectory)"
 
```

Comment:
Should you mention about the publishing of results.. I feel like I am left hanging at the moment

Reply:
It mentions about the publishing of the results in the section above :)